app/src/androidTest/java/com/infomaniak/drive/KDriveTest.kt

```diff
@@ -82,7 +83,10 @@ open class KDriveTest {
             okHttpClient = runBlocking { KDriveHttpClient.getHttpClient(user.id) }
 
             setUpRealm()
-            grantPermissions(android.Manifest.permission.WRITE_EXTERNAL_STORAGE)
+
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
+                grantPermissions(android.Manifest.permission.WRITE_EXTERNAL_STORAGE)
+            }
```

Comment:
And above tiramisu we do nothing?

You have to add `READ_MEDIA_IMAGES` and `READ_MEDIA_VIDEO`
https://github.com/Infomaniak/android-kDrive/blob/81131ee296b02d06b980ed4c15421e1b465f9f8a/app/src/main/java/com/infomaniak/drive/utils/DrivePermissions.kt#L163